src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -1,9 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+import { errorRedirect } from "../../../modules/helpers/errorRedirect";
+import { getSubmittedDocument } from "./PackageReviewService";
 import "./DocumentList.css";
 
-export default function DocumentList({ documents }) {
+export default function DocumentList({ packageId, documents }) {
+  function handleClickFile(document) {
+    getSubmittedDocument(packageId, document).catch((err) => {
+      errorRedirect(sessionStorage.getItem("errorUrl"), err);
+    });
+  }
+
+  function handleKeyDownFile(e, document) {
+    if (e && e.keyCode === 13) {
+      getSubmittedDocument(packageId, document).catch((err) => {
+        errorRedirect(sessionStorage.getItem("errorUrl"), err);
```

Comment:
Duplicate of line 11, not a big deal...

Reply:
Yeah, we need to come up with a better way for error handling globally.  Just redirecting to the error view is not always appropriate.  We instead should show a toast error banner, or have a reserved notification area, or ...

Reply:
Yes let's add one for it